modules/ROOT/pages/_partials/partner-and-customer-guide.adoc

```diff
@@ -144,10 +165,29 @@ See xref:java-support.adoc#mule-runtime[Mule runtime] to learn more about the Ja
 See xref:general::java-support.adoc#mule-runtime[Mule runtime] to learn more about the Java 17 changes made in Mule runtime engine. 
 +
 ** Reflection on Java classes
++
+To ensure compatibility with Java 17, review and update any code that uses reflection to access Java classes or set parameter values. Java 17 imposes stricter access controls, so you should avoid introspecting JDK internals. Modify your code to eliminate deep introspection of JDK classes and ensure that reflective access is limited to necessary areas only.
```

Comment:
wasn't this already mentioned above?

Reply:
Yes, you can ignore the repeated info. It is a CX thing and won't appear duplicate on the docs site. I will replace the below info with the feedback later